tests/internals.rs

```diff
@@ -137,8 +142,9 @@ fn create_account_tx_test_state(
             for (contract_address, class_hash) in address_to_class_hash {
                 let storage_keys = storage_view
                     .iter()
-                    .filter_map(|((k0, k1), v)| {
-                        (k0 == &contract_address).then_some((k1.clone(), v.clone()))
+                    .filter_map(|((address, storage_key), storage_value)| {
+                        (address == &contract_address)
+                            .then_some((storage_key.clone(), storage_value.clone()))
```

Comment:
This will `.clone()` both `storage_key` and `storage_value` whether they're used or not. Maybe using `.then(|| (storage_key.clone(), storage_value.clone()))` would be better?